services/cd-service/pkg/fs/fs.go

```diff
@@ -484,7 +484,8 @@ func (t *TreeBuilderFS) MkdirAll(dir string, perm os.FileMode) error {
 }
 
 func (t *TreeBuilderFS) Lstat(path string) (os.FileInfo, error) {
-	return nil, billy.ErrNotSupported
+	// TODO(HVG): implement this to support actual symlinks
```

Comment:
when will this TODO be done? Please add a reference to a story that handles it (or do it now)

Reply:
Yes, I will!

Reply:
I can probably do it in the same story but I will put it in a seperate PR.